### ISSUE TITLE
Support legacy Github handles as LaminHub handles

### ISSUE THREAD

**Issue by ap--** [laminlabs/laminhub-public#5]
### Report

Hello Lamin Team,

I signed up on lamin.ai using my github account, and my GitHub username `"ap--"` is an old legacy handle. (The handle would be unsupported when signing up today)

I could not use this handle for lamin.

I'm just reporting this here, so you are aware of it. I'm not sure if you'd consider this a bug or not, but it would be nice to be able to see the restrictions placed on lamin user handles.

Cheers,
Andreas 😃

**Comment by falexwolf** [comment]
Good point! I wasn't aware that `ap--` wouldn't be allowed (neither on lamin nor github).

**Comment by falexwolf** [comment]
You can change to your GitHub handle upon the next prod deploy, @ap--!
